tests/test_viv_features.py

```diff
@@ -6,336 +6,21 @@
 #  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
```

Comment:
as you can see, backend feature tests become much simpler.

Reply:
oh yeah 😄